auth/keycloak/config/realm-export.json

```diff
@@ -791,14 +791,14 @@
     "id" : "89ff6cf4-3755-4329-adab-ccfb74052c97",
     "clientId" : "tno-app",
     "name" : "Today's News Online",
-    "rootUrl" : "http://localhost:50080/app",
-    "baseUrl" : "http://localhost:50080/app",
+    "rootUrl" : "http://localhost:40080/app",
```

Comment:
Fixed port in keycloak configuration.  You can manually update your local instance or you will need to reinitialize your keycloak container and database.